### PR TITLE
Hey I added DFS java program .

### PR DESCRIPTION
**Pull request by rajshree2000**
I hope this helps and thank you so much for this repository , this is my first time contributing to open source for hacktoberfest. Please do consider my contribution.

### CODE REVIEW

File: Task 3/DFS [hunk 1]
```diff
@@ -0,0 +1,85 @@
+##Depth First Search (DFS) Java Program
```
Comment (nimishbongale) [inline Task 3/DFS:1]: This file should be under a folder names DFS.

Something like `Task 3/DFS/DFS.java`

**Review by nimishbongale — COMMENTED** [review]
Great work @rajshree2000!

It's nearly done. One small change. Let's get this merged!

File: Task 3/DFS (hunk 1 above)
Comment (rajshree2000) [reply Task 3/DFS:1]: Hey, I tried making changes but it's showing  "Sorry, a file exists where you’re trying to create a subdirectory. Choose a new path and try again". Can you please help me out

Comment (nimishbongale) [reply Task 3/DFS:1]: Let me have a look!

File: Task 3/DepthFirstSearch/DepthFirstSearch.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
+##Depth First Search (DFS) Java Program
+To traverse in trees we have traversal algorithms like inorder, preorder, postorder. Same way to traverse in graphs we have mainly two types of algorithms called DFS (Depth First Search) and BFS (Breadth First Search).
+In Depth First Search traversal we try to go away from starting vertex into the graph as deep as possible. We may face the case that our search never ends because, unlike tree graph may contains loops. Since this reason we maintain a Boolean array which stores whether the node is visited or not.
+
+Below program shows implementation of dfs in Java.
```
Comment (nimishbongale) [inline Task 3/DepthFirstSearch/DepthFirstSearch.java:5]: I've resolved the previous conflict. Please change this to a multiline comment. 

Something like 

```java
/*
insert 
description
here
*/

File: Task 1/FindTheLeader(Prepbytes)/solution.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import java.util.*;
+  import java.io.*;
+  
+  public class Main {
+    public static void main(String args[]) throws IOException {
+       Scanner sc = new Scanner(System.in);
+       int t = sc.nextInt(); //for testcase
+        while(t-->0){
+            int n = sc.nextInt();
```
Comment (nimishbongale) [inline Task 1/FindTheLeader(Prepbytes)/solution.java:9]: Can the code indentation be made a bit better here?

File: Task 1/FindTheLeader(Prepbytes)/solution.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            for(int i = n-2;i>=0;i--)
+                if(arr[i] >= max_from_right) {  //to check to find out all the elements that are greater than or equal to all the elements to it's right side.
+                    max_from_right = arr[i];
+                    System.out.println(max_from_right + " ");
+                }
+
+        }
+       
+       
+
```
Comment (nimishbongale) [inline Task 1/FindTheLeader(Prepbytes)/solution.java:24]: You can remove the unnecessary tabs/spaces here 👍

File: Task 3/DepthFirstSearch/DepthFirstSearch.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        obj.addEdgesToGraph(3,4);
+        obj.addEdgesToGraph(4,3);
+        
+        obj.DFSearch(1);
+    }
+}
+
+##Output
+
+1 2 5 6 4 7 8 3 9
```
Comment (nimishbongale) [inline Task 3/DepthFirstSearch/DepthFirstSearch.java:85]: Add this as a part of the multiline comment on the top

**Review by nimishbongale — COMMENTED** [review]
Good job @rajshree2000!

Suggested small changes just to make sure everything is as per the expected formatting. I'm sure you can do them!

**Comment by rajshree2000** [comment]
I've made all the changes. Can you please review if it's ok or not.

**Review by nimishbongale — APPROVED** [review]
Great job @rajshree2000! 🚀 

You are now ready to contribute to cool open source projects! 😄

**Comment by rajshree2000** [comment]
Thank you so much, it means a lot. Thank you for helping me out and for this repository. Learned a lot.

**Pull request merged**